packages/@ember/routing/router.ts

```diff
@@ -1 +1,1933 @@
-export { Router as default } from '@ember/-internals/routing';
+import { privatize as P } from '@ember/-internals/container';
+import type { OutletState as GlimmerOutletState, OutletView } from '@ember/-internals/glimmer';
+import { computed, get, set } from '@ember/object';
+import type { Factory, FactoryClass, Owner } from '@ember/-internals/owner';
+import { getOwner } from '@ember/-internals/owner';
+import { BucketCache, DSL } from '@ember/routing/internals';
+import type { DSLCallback, EngineRouteInfo } from '@ember/routing/internals';
+import { RouterState } from '@ember/routing/internals';
+import {
+  calculateCacheKey,
+  extractRouteArgs,
+  getActiveTargetName,
+  resemblesURL,
+} from './lib/utils';
+import type { RouteArgs, RouteOptions } from './lib/utils';
+import EmberLocation from '@ember/routing/location';
+import type { ILocation as IEmberLocation } from '@ember/routing/location';
```

Comment:
Another case where the imports should be able to be merged on a recent-enough version of TS. And I checked just now: it's actually 4.5, and supports this—

```suggestion
import EmberLocation, type { ILocation as IEmberLocation } from '@ember/routing/location';
```

(Distinctly not a blocker of course.)

Reply:
Hmm... I'm getting syntax errors and we're on 4.6.

Reply:
Yeah, for posterity, the correct syntax is:

```ts
import EmberLocation, { type ILocation as IEmberLocation } from '@ember/routing/location';
```